spec/core/hash/transform_keys_spec.rb

```diff
@@ -76,7 +76,7 @@
   end
 
   ruby_version_is "3.0.2" do
-    it "returns the processed keys and non evaluated keys if we break from the block" do
+    xit "returns the processed keys and non evaluated keys if we break from the block" do
```

Comment:
I think we should put `NATFIXME` notes here so we don't forget to come back and fix these. (Unless of course, you already have fixes on the way... :wink:)

Reply:
Thanks for catching that! We should definitely do that. I'm going to add them later today.